spec/pushmi_pullyu/swift_depositer_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'spec_helper'
+
+RSpec.describe PushmiPullyu::SwiftDepositer do
+  it 'not valid without arguments' do
+    expect { described_class.new }.to raise_error(ArgumentError)
```

Comment:
This is just testing that Ruby isn't broken. You should assume that Ruby works as Ruby does, and not write tests to validate that the universe isn't broken.